app/chat/src/frontendMain/kotlin/main.kt

```diff
@@ -0,0 +1,100 @@
+package io.ktor.samples.chat.frontend
+
+import io.ktor.client.HttpClient
+import io.ktor.client.features.websocket.WebSocketException
+import io.ktor.client.features.websocket.WebSockets
+import io.ktor.client.features.websocket.webSocketSession
+import io.ktor.http.HttpMethod
+import io.ktor.http.cio.websocket.Frame
+import io.ktor.http.cio.websocket.WebSocketSession
+import io.ktor.http.cio.websocket.readText
+import io.ktor.util.KtorExperimentalAPI
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.channels.ClosedReceiveChannelException
+import kotlinx.coroutines.launch
+import org.w3c.dom.HTMLElement
+import org.w3c.dom.HTMLInputElement
+import org.w3c.dom.events.KeyboardEvent
+import kotlin.browser.document
+import kotlin.browser.window
+
+@KtorExperimentalAPI
```

Comment:
=> `UseExperimental` or `OptIn`

Reply:
`UseExperimental` is deprecated. If I change `KtorExperimentalAPI` to `OptIn` then the IDE shows me a warning from an intention: 
```
[EXPERIMENTAL_API_USAGE] This API is experimental. It could be removed or changed in future releases or it's behaviour may be different.
```